homeassistant/components/blueprint/models.py

```diff
@@ -86,6 +86,10 @@ def update_metadata(self, *, source_url: Optional[str] = None) -> None:
         if source_url is not None:
             self.data[CONF_BLUEPRINT][CONF_SOURCE_URL] = source_url
 
+    def dump(self) -> str:
```

Comment:
Shouldn't we call this `yaml`  ? Something can be dumped to any format.

Reply:
True, `dump_yaml`?